examples/complete-kubernetes-addons/main.tf

```diff
@@ -108,18 +108,19 @@ module "eks_blueprints" {
 
   fargate_profiles = {
     default = {
-      fargate_profile_name = "default"
-      fargate_profile_namespaces = [
+      name = "default"
+      selectors = [
         {
           namespace = "default"
-          k8s_labels = {
+          labels = {
             Environment = "preprod"
             Zone        = "dev"
             env         = "fargate"
           }
-      }]
-      subnet_ids = module.vpc.private_subnets
```

Comment:
I assume you removed it before the default behavior is to use the same private subnets as provided to the cluster, do we think it may be beneficial to show how to use specific subnets?

Reply:
I would say - it depends. what do we want to show here under this example thats called "complete-kubernetes-addons"?

Reply:
> I would say - it depends. what do we want to show here under this example thats called "complete-kubernetes-addons"?

Fair, maybe we should consider adding this in the appropriate [example](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/cbc31b0bbb2c859681a2e456f13850ec5bad260b/examples/fargate-serverless/main.tf#L61-L91) instead, WDYT?

Reply:
I think there are two avenues here that we are mixing
1. How to use the `terraform-aws-eks` module
2. How to achieve an outcome

For one, we can point to the appropriate location that demonstrates all the different ways the underlying module can be configured https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/examples/fargate_profile/main.tf

Point 2 is what I think we are mostly concerned with here (IMO). If someone looks at the `serverless-fargate` example and sees the different way of specifying subnets - does that mean thats how they have to be configured to create a serverless cluster running Fargate?